CMakeLists.txt

```diff
@@ -55,7 +55,7 @@ SET(SOURCES src/ctrl.c src/ctrl-gen.c src/device.c src/diag.c
 include_directories(
   ${libuvc_SOURCE_DIR}/include
   ${libuvc_BINARY_DIR}/include
-  ${LIBUSB_INCLUDE_DIR}
+  ${LIBUSB_INCLUDEDIR}
```

Comment:
Why is it `LIBUSB_INCLUDEDIR` here? This is not standard cmake naming.
Looks like a typo to me actually especially since we use `LIBUSB_INCLUDE_DIR` in other places in the `CMakeLists.txt`.